### PR TITLE
Add a table for screenshots to the pull request template

### PR DESCRIPTION
**Pull request by mplorentz**
Just adding something to the template I often do. Anyone else?

### CODE REVIEW

**Review by bryanmontz — APPROVED** [review]
I use markdown tables in just about every PR. Thanks!

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -10,3 +10,7 @@ A description of the changes proposed in the pull request. Explain what it does
 
 ## Screenshots/Video
 Post screenshots or video showing your changes, ideally showing how the app worked before and after these changes. Delete this section if this PR contains no visual changes.
+
+| Before | After |
+|--------|--------|
+| (replaceme) | (replaceme) |
```
Comment (joshuatbrown) [inline .github/pull_request_template.md:16]: This feels like a ridiculous comment to make, but I'll make it anyway. Could we remove the parentheses here? I often double-click to select the text to replace, and double-clicking "replaceme" selects only those characters and not the parentheses. So then it's another step to delete those. It's not a big deal, but it'd be nice to adjust this to make it slightly easier.

```suggestion
| replaceme | replaceme |
```

**Review by joshuatbrown — COMMENTED** [review]
Love the idea here.

File: .github/pull_request_template.md (hunk 1 above)
Comment (mplorentz) [reply .github/pull_request_template.md:16]: This is a very fair and good nitpick 👌

**Review by joshuatbrown — COMMENTED** [review]
💯

**Review by joshuatbrown — APPROVED** [review]
🏆

**Pull request merged**